### ISSUE TITLE
To become empty standard-input when sending selected text to UNIX script.

### ISSUE THREAD

**Issue by knlab** [coteditor/CotEditor#760]
## Environment

- CotEditor: 3.2.7 (212)
- System: macOS バージョン10.13.2（ビルド17C205）
- Language: Japanese

## Short Description

When calling UNIX script from the script menu then often get a empty standard-input from Cot-Editor.

It seems can not standard-input be send for scripts when repeat executions.

## Steps to Reproduce the Issue
Sample-code is:

```php
#! /usr/bin/php
<?php
// %%%{CotEditorXInput=Selection}%%%
// %%%{CotEditorXOutput=ReplaceSelection}%%%
$stdin = file_get_contents("php://stdin");
$stdout = '<' . $stdin . '/>';
echo $stdout;
?>
```

For example, when selecting a string on the cot-editor then expect to convert to tags `<selected-strings/>`. But, the result that is often output with `</>`.

Maybe, this problem didn't occur on macOS 10.12.
Also, It seem a problem does not occur when assigning a keybord shortcut to the script menu.

**Comment by 1024jp** [comment]
cf. #733, #739

**Comment by 1024jp** [comment]
Thank you for the report.
I've just reproduced your issue under my environment. 
I'll try to fix this on CotEditor 3.3.0.

---

[dev note] The issue may occur because the `outPipe` is released after the task execution and therefore `writeabilityHandler(_:)` is not performed.
